omnibus/config/patches/libselinux/ln_no_relative.patch

```diff
@@ -1,14 +1,14 @@
 --- a/src/Makefile
 +++ b/src/Makefile
-@@ -178,12 +178,12 @@
+@@ -184,12 +184,12 @@
  	install -m 755 $(LIBSO) $(DESTDIR)$(SHLIBDIR)
  	test -d $(DESTDIR)$(LIBDIR)/pkgconfig || install -m 755 -d $(DESTDIR)$(LIBDIR)/pkgconfig
  	install -m 644 $(LIBPC) $(DESTDIR)$(LIBDIR)/pkgconfig
 -	ln -sf --relative $(DESTDIR)$(SHLIBDIR)/$(LIBSO) $(DESTDIR)$(LIBDIR)/$(TARGET)
 +	ln -sf $(DESTDIR)$(SHLIBDIR)/$(LIBSO) $(DESTDIR)$(LIBDIR)/$(TARGET)
  
  install-pywrap: pywrap
- 	$(PYTHON) setup.py install --prefix=$(PREFIX) `test -n "$(DESTDIR)" && echo --root $(DESTDIR)` $(PYTHON_SETUP_ARGS)
+ 	$(PYTHON) -m pip install --prefix=$(PREFIX) `test -n "$(DESTDIR)" && echo --root $(DESTDIR) --ignore-installed --no-deps` $(PYTHON_SETUP_ARGS) .
```

Comment:
Is this an upstream patch? Otherwise could you clarify the intent?

Reply:
It's a custom patch. We had to do this, because our build images use a very old version of GNU Coreutils, where `ln` doesn't support the `--relative` parameter.

```
$ ln -sf --relative /opt/datadog-agent/embedded/lib/libsepol.so.2 /opt/datadog-agent/embedded//lib/libsepol.so
ln: unrecognized option '--relative'
Try `ln --help' for more information.
```

Reply:
Fair enough for the `ln` part, I was mostly thinking about switching from `setup.py` to `-m pip` though (then again, not a blocking question)

Reply:
This part is an upstream change from `libselinux`. It appears in the diff context. It's not actually part of the diff changes.

Reply:
Oh my bad, I missed the different indentation level. Thanks for the clarification!